lib/content-services/src/lib/document-list/components/document-list.component.ts

```diff
@@ -466,13 +467,13 @@ export class DocumentListComponent extends DataTableSchema implements OnInit, On
         private dialog: MatDialog
     ) {
         super(appConfig, 'default', presetsDefaultModel);
-        this.nodeService.nodeUpdated.pipe(takeUntil(this.onDestroy$)).subscribe((node) => {
+        this.nodeService.nodeUpdated.pipe(takeUntilDestroyed(this.destroyRef)).subscribe((node) => {
```

Comment:
I think no need to pass destroyRef to takeUntilDestroyed here because it's used inside constructor. The takeUntilDestroyed can be used without destroyRef